cypress/e2e/functional/opal/manualAccountCreation/adultOrYouthOnly/PO-358_contactDetails.feature

```diff
@@ -133,17 +151,19 @@ Feature: PO-358_Contact Details for adult or youth only
     Then I click on "Contact details" link
     Then I see "Contact details" on the page header
 
-    Then I verify "<primaryEmail>","<secondaryEmail>","<mobileTelephone>","<homeTelephone>","<businessTelephone>" on contact details page
+    Then I verify "<updatePrimaryEmail>","<updateSecondaryEmail>","<updateMobileTelephone>","<updateHomeTelephone>","<updateWprkTelephone>" on contact details page
```

Comment:
generic steps can be used

Reply:
In generic steps passed directly value here I'm passing as a variable